src/test/java/com/google/cloud/tools/maven/deploy/AbstractDeployMojoTest.java

```diff
@@ -59,9 +55,9 @@ public void testGetProjectId_onlyProject() throws IllegalAccessException {
   }
 
   @Test
-  public void testGetProjectId_onlyProjectId() throws IllegalAccessException {
-    projectField.set(testMojo, "someProject");
-    testMojo.setProjectId("someProjectId");
+  public void testGetProjectId_onlyProjectId() {
```

Comment:
I think you mixed up the names of these two methods (`onlyProjectId` and `bothProjectAndProjectId`)